guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -7,6 +7,9 @@
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
+// If this section would be empty otherwise, uncomment the following line:
+// There are no upgrade warnings with Foreman {ProjectVersion}.
```

Comment:
```suggestion
There are no upgrade warnings with Foreman {ProjectVersion}.
```

I'd say keep it uncommented. The first person to add a release note can then remove the line.

Reply:
There is an upgrade warning in line 13.

Reply:
Right, I missed that. Sorry.